app/presenters/summary/html_presenter.rb

```diff
@@ -142,9 +143,11 @@ def capital_sections
 
     private
 
+    attr_reader :editable
+
     def build_sections(relevant_sections)
       SECTIONS.fetch(relevant_sections).map do |section|
-        Sections.const_get(section.to_s.camelize).new(crime_application)
+        Sections.const_get(section.to_s.camelize).new(crime_application, editable:)
       end.select(&:show?)
```

Comment:
☝️ required to hide the change links when showing a draft applications to CL providers.